apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-detail.full-view.presenter.tsx

```diff
@@ -68,6 +69,13 @@ const filterEntriesForSelectedTerms = (
         return doValuesMatchFilters(variableValues, filter);
     });
 
+const TermPresenterForVocabularyListEntry = (
+    props: ICategorizableDetailQueryResult<ITermViewModel>
+): JSX.Element => (
+    // TODO Write a custom presenter here
```

Comment:
Oh we should do this now. Note that the back-end actually provides a distinct view model type for `IVocabularyListEntry['term']` now thanks to recent work I've done. Justin and I can probably take a look at this some time.